.nomad/scripts/set-job-name.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sed -i "/job/c\job \"$1\" {" apiary.nomad
```

Comment:
What does this do? A comment might be helpful here.

Reply:
It is not currently possible to set the job ID dynamically: https://github.com/hashicorp/nomad/issues/9522

This script is a quick way to replace the job ID on disk before submitting the job. It is not used for anything yet, but will be integrated into the CD jobs at some point.